packages/core/codeframe/src/codeframe.js

```diff
@@ -170,13 +179,24 @@ export default function codeFrame(
 
           let whitespaceLength = startCol - lastCol;
           if (whitespaceLength > 0) {
-            highlightLine += ' '.repeat(whitespaceLength);
+            let whitespace = originalLine
+              .substring(lastCol, startCol)
```

Comment:
Any particular reason why `lastCol` and `startCol` are swapped here (substring works fine, but it still looks kinda strange)?